src/applications/mhv/secure-messaging/components/ThreadList/ThreadListItem.jsx

```diff
@@ -121,10 +132,27 @@ const ThreadListItem = props => {
           className="message-subject-link vads-u-margin-y--0p5"
           to={`${Paths.MESSAGE_THREAD}${messageId}/`}
         >
-          {categoryLabel}: {getHighlightedText(subject)}
+          {hasAttachment ? (
+            <span id={`message-link-has-attachment-${messageId}`}>
+              {categoryLabel}: {getHighlightedText(subject)}
+              <span className="sr-only">Has attachment</span>
+            </span>
+          ) : (
+            <span id={`message-link-${messageId}`}>
+              {categoryLabel}: {getHighlightedText(subject)}
+            </span>
+          )}
         </Link>
+
         <p className="received-date vads-u-margin-y--0p5">
-          {hasAttachment && <i className={attachmentClasses} aria-hidden />}
+          {hasAttachment && (
+            <span
+              role="img"
+              aria-labelledby={`message-link-has-attachment-${messageId}`}
+            >
+              <i className={attachmentClasses} alt="Has attachment" />
```

Comment:
Just flagging if role="img" needs to be added here as well as in instance above 🤷🏻‍♂️